macros/base/start_print.cfg

```
@@ -369,7 +371,8 @@ gcode:
     # ----- BED MESH -------------------------------------------
     {% set FL_SIZE = printer["gcode_macro START_PRINT"].fl_size %}
     {% set BED_MESH_PROFILE = printer["gcode_macro START_PRINT"].bed_mesh_profile %}
-
+    {% set force_mesh = printer["gcode_macro START_PRINT"].force_mesh %}
```

Please use uppercase for the var name here to keep it consistent:

```suggestion
    {% set FORCE_MESH = printer["gcode_macro START_PRINT"].force_mesh %}
```